Loop/Managers/Alerts/AlertManager.swift

```diff
@@ -251,6 +251,33 @@ extension AlertManager {
     }
 }
 
+// MARK: PersistedAlertStore
+extension AlertManager: PersistedAlertStore {
+    public func lookupOutstandingAlerts(managerIdentifier: String, completion: @escaping (Result<[PersistedAlert], Error>) -> Void) {
+        alertStore.lookupAllUnretracted(managerIdentifier: managerIdentifier) {
```

Comment:
thinking about the use case I have for coastal integration, I would want to know all alerts that have been issued but unacknowledged and unretracted to avoid issuing the same alert again. Could this be added as well?